witness/src/Dscp/Witness/Logic/Apply.hs

```diff
@@ -1,15 +1,18 @@
--- | Creating and application.
+-- | Block application.
 
-module Dscp.Witness.Logic.Processing
-    ( createPayload
-    , createBlock
+module Dscp.Witness.Logic.Apply
+    ( applyBlockRaw
+    , applyBlockRawPure
     , applyBlock
+    , applyBlockPure
     , applyGenesisBlock
+    , applyGenesisBlockPure
     ) where
 
 import Data.Default (def)
 import qualified Data.Map as M
 import Data.Time.Clock (UTCTime (..))
+
```

Comment:
???

Reply:
`-- TODO exceptions`: will make an issue now.

> Is it not going to change?
It is not going to, I tried and this brings problems.

> ???
I've separated imports from stdlib and imports from lootbox/SD.

Reply:
@volhovm there you are: https://issues.serokell.io/issue/DSCP-291

Reply:
>I've separated imports from stdlib and imports from lootbox/SD.

That's my point, I find it useless. You usually have 2 blocks of imports + one line for prelude if it's needed.

Reply:
Also refer to DSCP-291 in the comment or erase it.